dispatcher_plugin_nb2workflow/queries.py

```diff
@@ -32,10 +32,13 @@ def construct_parameter_lists(backend_param_dict, ontology_path):
                             "http://odahub.io/ontology#StartTime": "T1",
                             "http://odahub.io/ontology#EndTime": "T2",
                             "http://odahub.io/ontology#AstrophysicalObject": "src_name",
-                            "ThisNameShouldNotExist": "token"
+                            "ThisNameShouldNotExist": "token",
                           }
-    par_name_substitution = {}
-
+    par_name_substitution = {'token': '_token'}
+    # There are two name substitutions for "*token": 
+    # 1) if backend notebook defines parameter named "token", it will be renamed by general mechanism (appending "_rename")
```

Comment:
where is this renaming taking place ?

Reply:
`par_name_substitution` dict gets populated lower depending on conditions (either rename or keep the name) for all parameters.

Then, it gets used at L116-117 to pass proper names to backend